src/components/feedbacks/list/FeedbackItem.tsx

```diff
@@ -1,14 +1,19 @@
 import styles from '../Feedbacks.module.scss'
 import { FC } from 'react'
 import { Rating } from './Rating'
-
+import { Review } from '../../../redux/feedbacks/slice'
 type Props = {
-  reviews: []
+  review: Review
 }
 
-// TODO: remove this comment
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const FeedbackItem: FC<Props> = ({ reviews }) => {
+const FeedbackItem: FC<Props> = ({ review }) => {
+  const date = new Date(review.created_at)
+  const day = date.getDate()
+  const month = date.getMonth() + 1
+  const year = date.getFullYear()
+
+  const formattedDate = `${day < 10 ? '0' : ''}${day}.${month < 10 ? '0' : ''}${month}.${year}`
```

Comment:
It would be better if it were a function separately so that it could be reused and would be more readable